common/lib/plugin_service.ts

```diff
@@ -332,7 +332,7 @@ export class PluginServiceImpl implements PluginService, HostListProviderService
   }
 
   isBlockingHostListProvider(arg: any): arg is BlockingHostListProvider {
-    return arg;
+    return arg != null && typeof arg.clearAll === "function" && typeof arg.forceMonitoringRefresh === "function";
```

Comment:
Should we store things like "function" or "string" as constants somewhere?

Reply:
For this scenario this worked, but if i need to use the value later on, like in this scenario
```
const sql = typeof queryTextOrConfig === ConstantTypes.STRING_TYPE ? queryTextOrConfig : queryTextOrConfig.text;
```
There will be an error 
<img width="1131" height="138" alt="image" src="https://github.com/user-attachments/assets/bd5e4229-bd35-4ee0-b762-c47c9b35833f" />
I will just keep it as "string" for now.